pandas/tests/indexes/conftest.py

```diff
@@ -1,8 +1,30 @@
+import numpy as np
 import pytest
 
 import pandas._testing as tm
 from pandas.core.indexes.api import Index, MultiIndex
 
+
+def _gen_mi():
+    # a MultiIndex used to test the general functionality of the
+    # general functionality of this object
+
+    # See Also: tests.multi.conftest.idx
+    major_axis = Index(["foo", "bar", "baz", "qux"])
```

Comment:
Can you alternately just construct the mi from tuples? I think would be more readable that way

Reply:
This is how it is constructd in the other conftest